docs/api.md

```
@@ -678,6 +680,97 @@ POST /api/interviews/{interviewId}/sessions/finish
 
 ---
 
+### 종합 리포트 조회
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)